src/Interpreters/Cache/FileSegment.cpp

```diff
@@ -1121,9 +1121,12 @@ void FileSegment::increasePriority()
     if (it)
     {
         if (auto cache_lock = cache->tryLockCache())
-            hits_count = it->increasePriority(cache_lock);
+            it->increasePriority(cache_lock);
```

Comment:
Makes sense to change the return type to `void`, then?

Reply:
I'll do that later) It would require changes to private as well, so I'd avoid it, because of easier backporting